config.ru

```diff
@@ -1,8 +1,8 @@
-# frozen_string_literal:true
+# This file is used by Rack-based servers to start the application.
+require_relative 'config/environment'
```

Comment:
This needs to be loaded after Honeycomb, your PR shouldn't need to affect this file.

Reply:
This happened on the rebase. Ill update.